hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/upgrade/SCMUpgradeFinalizer.java

```diff
@@ -74,33 +70,107 @@ public boolean preFinalizeUpgrade() throws IOException {
     msg += "\n  New pipelines creation will remain frozen until Upgrade " +
         "is finalized.";
 
-    storageContainerManager.preFinalizeUpgrade();
+    PipelineManager pipelineManager = scm.getPipelineManager();
+    /**
```

Comment:
Can we move this to be the apidoc of the called method instead?

Reply:
Fixed